CONTRIBUTING.md

```diff
@@ -7,9 +7,9 @@ into a single location:
 **Quick links**:
 * [Docs help](https://knative.dev/help/contributor/)
     * New content templates:
-        * [Concept](./docs/contributor/templates/template-concept.md)
-        * [Procedure](./docs/contributor/templates/template-procedure.md)
-        * [Troubleshooting](./docs/contributor/templates/template-troubleshooting.md)
-        * [Blog](./docs/contributor/templates/template-blog-entry.md)
+        * [Concept](docs/help/contributor/templates/template-concept.md)
+        * [Procedure](docs/help/contributor/templates/template-procedure.md)
+        * [Troubleshooting](docs/help/contributor/templates/template-troubleshooting.md)
+        * [Blog](docs/help/contributor/templates/template-blog-entry.md)
 * [Website help](https://knative.dev/help/contributor/publishing)
 * [Maintainer help](https://knative.dev/help/maintainer/)
```

Comment:
These links don't work, is there a new version of these pages?

Reply:
I've removed them and I'll make a separate issue for linking to the old (and currently somewhat out-of-date) maintainer topics.